### DISCUSSION TITLE
Welcome to Zavala Discussions!

### DISCUSSION THREAD

**Discussion by vincode-io** [vincode-io/Zavala#52] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we build together 💪.

  To get started, comment below with an introduction of yourself and tell us what you like about outliners.

**Comment by jacobsamlarose** [comment]
Hi! Interested in seeing how this will develop. OmniOutliner provided my first introduction to outliner apps. Since then I've moved through Workflowy and Dynalist, and I'm currently experimenting with Roam. As it currently stands, iOS is my primary computing platform (iPad Pro + Mini + iPhone 8); happy to offer thoughts and feedback when mobile apps are available to test.

**Comment by vincode-io** [comment]
@jacobsamlarose Thanks for the interest.  I'll be sure to let everyone know when a TestFlight version is available for testing.

**Comment by macosxguru** [comment]
I use [Opal](http://a-sharp.com/opal/), [Drafts](https://getdrafts.com) and [TaskPaper](https://www.taskpaper.com) for my outlining needs.

My mind works best in an outline and I have been using outliners for more than thirty years to help with my thinking. 

The closest product to Zavala that I can think of is [Outlinely](https://apps.apple.com/us/app/outlinely-outline-notes/id843692954?mt=12). It is a library based outliner. It is a good product but development is sporadic and unreliable. Zavala would be a good addition to the arsenal of tools available to the writer on macOS or iOS. 

macosxguru
---
Email: macosxguru@gmail.com
Website: http://bicycleforyourmind.com
Twitter: http://twitter.com/macosxguru

**Comment by latvia234** [comment]
Via [this topic](https://www.outlinersoftware.com/topics/viewt/9262/0/zavala--an-open-source-outliner-for-macs-ipads-and-iphones) I found Zavala. Thanks for creating that topic.

Around 2012 I discovered Workflowy, there was a time when Workflowy development was not visible. I did miss some options in Workflowy that I found in Dynalist. So that is when I started to use Dynalist.

Since a few months I'm thinking more and more about moving all my outlines to my own computer instead of keeping them on a server from Dynalist.

Because of that I found and tested a lot of tools including Opal, TaskPaper, Outlinely that macosxguru mentioned. But I also tried OmniOutliner, [Obsidian+Obsidian outliner](https://github.com/vslinko/obsidian-outliner), [Athens](https://github.com/athensresearch/athens) and [Logseq](https://github.com/logseq/logseq).

I am happy to see that Zavala is available and hope that I can use it as my main outliner soon.

How can we let more people know about Zavala? This project deserves more active users. :)

**Comment by pooldoo** [comment]
I’ve been a fan of outliners since ECCO Pro in the mid 1990’s (yeah, I’m old.)  ECCO was solid as a rock, full featured, and easy to use.  I’ve been unhappy with the offerings since. I am super excited about this new project. I’m enjoying the minimalist approach.

**Comment by vincode-io** [comment]
Welcome @pooldoo! If I can keep that minimalist feel and still add features, I think I've succeeded in my design goals.

Please file bugs you find and features you would like to see in GitHub Issues. Bugs especially.
